Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM onsdigital/flask-crypto
 
 ADD consumer.py /app/consumer.py
+ADD async_consumer.py /app/async_consumer.py
```

Comment:
We probably should as a general rule, put the app's code under an app folder from the main root of the project - that way we only need add a single folder in the dockerfile.